clairvoyance/graphql.py

```diff
@@ -12,7 +13,8 @@
 from typing import Set
 
 
-def post(url, data=None, json=None, **kwargs):
+def post(url, data=None, json=None, proxies=None, **kwargs):
```

Comment:
I suppose the `proxies` can be passed as part of `**kwargs` so we don't need the explicit `proxies` parameter.

Reply:
The same way as `verify` is passed